.github/CONTRIBUTING.md

```diff
@@ -111,8 +111,8 @@ active development. The team's primary focus is on fixing known issues,
 improving performance, and adding compelling new features.
 
 You can view the list of proposed features
-by filtering the issue tracker by the ["Feature"
-label](https://github.com/zfsonlinux/zfs/issues?q=is%3Aopen+is%3Aissue+label%3AFeature).
+by filtering the issue tracker by the ["Type: Feature"
+label](https://github.com/zfsonlinux/zfs/issues?q=is%3Aopen+is%3Aissue+label%3A%22Type%3A+Feature%22).
```

Comment:
Thanks! Can you update these URLs to point to the openzfs/zfs repo, too.

Reply:
Let's tackle this in another PR.  I don't see any reason to delay merging this.